### PR TITLE
Use PyPI token instead of password

### PR DESCRIPTION
**Pull request by Mariatta**
Closes https://github.com/python/cherry-picker/issues/14

### CODE REVIEW

**Comment by Mariatta** [comment]
Where's travis? 😕

**Comment by encukou** [comment]
Travis is not enabled for this repository.
Could a repo admin click an "Activate" button at https://travis-ci.org/python/cherry-picker ?

**Comment by Mariatta** [comment]
We're using travis-ci.com and it was working a couple months ago
https://travis-ci.com/python/cherry-picker/

**Comment by encukou** [comment]
Ah! Sorry for the noise, then.

**Comment by webknjaz** [comment]
@Mariatta try closing and reopening the PR...

File: .travis.yml [hunk 1]
```diff
@@ -56,13 +56,11 @@ jobs:
       # This happenes when other CI (AppVeyor etc.) has already uploaded
       # the very same dist (usually sdist).
       skip-existing: true
-      user: &pypi-user core-workflow
+      user: &pypi-user @token
```
Comment (webknjaz) [inline .travis.yml]: Now it's
```suggestion
      user: &pypi-user __token__
```

**Comment by webknjaz** [comment]
I think it now makes sense to migrate to https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

**Comment by Mariatta** [comment]
Outdated PR.

**Pull request closed without merging**